src/report-export/services/codepen-report-export-service.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+
+import {
+    ExportFormat,
+    ExportFormProps,
+    ReportExportService,
+} from 'report-export/types/report-export-service';
+
+const CodePenReportExportServiceKey: ExportFormat = 'codepen';
+
+const exportForm = NamedFC<ExportFormProps>(
```

Comment:
The `exportForm` seems interesting enough that we should have unit tests for it.
The use of react hooks calls my attention here: currently on the repo, we don't use hooks so we don't have any example on how to unit tests this. It would be a very good contribution to write the first unit test around react hooks! If that seems to convoluted, there is always the possibility to go back to a more "traditional" react component so we can follow other examples in this repo on how to test it.